### ISSUE TITLE
Add "front-to-front" option for airflow.

### ISSUE THREAD

**Issue by jcollie** [netbox-community/netbox#8805]
### NetBox version

v3.1.8

### Feature type

Data model extension

### Proposed functionality

Add "front to front" option for airflow.

### Use case

The Cisco ISR4321 has its air exhaust and intake on the front, as seen in this diagram taken from a Cisco manual.

![Screenshot from 2022-03-06 07-54-27](https://user-images.githubusercontent.com/740022/156926435-b077958b-6a1e-4ed5-bff8-b1e859cd9f0b.png)

### Database changes

None.

### External dependencies

None.

**Comment by jeremystretch** [comment]
Ignoring the absurdity of this design, if a device takes in and exhausts air from the same face, there's no need to track airflow direction.

**Comment by jcollie** [comment]
Yes, it's a silly design IMHO but it exists and was a relatively popular router in it's day. If we can't have "front-to-front" can we get "other" to indicate that it's not a passively cooled device and that the airflow is not unknown (which None would indicate to me).

**Comment by jeremystretch** [comment]
I'm going to add an option for "mixed," which IMO should cover all such odd corner cases.
